angular/core/components/poll/count/chart_panel/poll_count_chart_panel.haml

```diff
@@ -6,5 +6,5 @@
       .poll-count-chart-panel__no{ng-style: "{'flex-basis': percentComplete(1), 'background-color': colors[1]}"}
       .poll-count-chart-panel__yes{ng-style: "{'flex-basis': percentComplete(0), 'background-color': colors[0]}"}
     .poll-count-chart-panel__data
-      .poll-count-chart-panel__numerator {{poll.stanceCounts[0]}}
+      .poll-count-chart-panel__numerator {{poll.stanceCounts[0] + poll.stanceCounts[1]}}
```

Comment:
I believe this could be poll.stancesCount?

Reply:
Happy to change it to that, yes, wasn't sure if we wanted to incorporate any future stances we may add such as acknowledged but undecided; will change now.